body.tex

```diff
@@ -300,29 +360,30 @@ \subsubsection{Defining Epics} \label{sec:planning-epics}
 \item The \texttt{Component} field set to the appropriate Department;
 \item The \texttt{Story Points} field set to a (non-zero) estimate of the effort required to complete the \gls{epic} in terms of \glspl{SP} (see \S\ref{sec:effort}).
 \item The \texttt{RO Milestone ID} field set to the appropriate L3 or L2 milestone that the epic will contribute to achieving (or achieve in its totality).
-  The exact \texttt{RO Milestone ID}text can be found in the Milestone issue or Smartsheet.
-\item The \texttt{Due Date} field set to the appropriate date, which does not exceed the due date of the Milestone it is labeled to achieve.
+  The exact \texttt{RO Milestone ID} can be found in the milestone issue or Smartsheet.
+\item The \texttt{Due Date} field set to the appropriate date, which does not exceed the due date of the milestone it is labeled to achieve.
 \item The \texttt{label} field is set to identify the fiscal year during which the work will be done. Examples are FY22 or FY23.
 
 \end{itemize}
 The fields above are required to have values entered because they define the connection to Smartsheet where effort-tracking for the full project is done. Other fields in the epic can also be filled in as needed.
 
-Be aware that: (Amanda: I don't think all of the following bullets are true or necessary in the Operations non-EVMS era.  Should clarify with Cathy and Wil.)
+Be aware that:
 
 \begin{itemize}
 \item An \gls{epic} may only be assigned to a single \gls{cycle}.
```

Comment:
We are not really doign cycles in ops and not cleanly like this. The epics have been event driven.

Reply:
Do you plan to move into a regime of cycle-driven work and construction ramps down?  That was an assumption I was working from.  We at least have the annual POP cycle that we will work within.